apollo/octopus/src/main/graphql/com/hedvig/android/apollo/octopus/graphql/claimflow/QueryEntryPointSearch.graphql

```diff
@@ -0,0 +1,6 @@
+query EntrypointSearch($input: EntrypointSearchInput) {
+  entrypointSearch(input: $input) {
+    displayName
+    id
+  }
+}
```

Comment:
Since I see that the input we give this in the code is always:
```
EntrypointSearchInput(
  type = EntrypointType.CLAIM,
  limit = Optional.present(NR_OF_ENTRYPOINTS),
)
```
So we can change this so that in the graphql file itself we just be
```suggestion
query EntrypointSearch {
  entrypointSearch(input: { type: CLAIM, limit: 20 }) {
    id
    displayName
  }
}
```
And we don't need to give it any more input.

If we ever need to change the limit, or add the search string we can adjust accordingly. Wdyt?